### PR TITLE
Adding running command in cluster utility to common

### PR DESCRIPTION
**Pull request by mshitrit**

### CODE REVIEW

File: pkg/command/command.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+	if err != nil {
+		return nil, err
+	}
+
+	exec, err := remotecommand.NewSPDYExecutor(cfg, "POST", req.URL())
+	if err != nil {
+		return nil, err
+	}
+
+	err = exec.Stream(remotecommand.StreamOptions{
```
Comment (clobrano) [inline pkg/command/command.go]: It seems that `exec.Stream` is deprecated
https://pkg.go.dev/k8s.io/client-go@v0.26.3/tools/remotecommand#Executor.Stream

**Review by clobrano — CHANGES_REQUESTED** [review]

**Comment by slintes** [comment]
/lgtm
/hold

I think Carlo was looking as well :)

**Comment by clobrano** [comment]
/lgtm 
Giving a chance to get more reviews, feel free to unhold
/hold

**Pull request merged**